build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -1061,6 +1066,16 @@ CONFIG_MTD_SMART_JOURNALING=y
 #
 # CONFIG_AIFW is not set
 
+#
+# CSI Framework
+#
+CONFIG_CSIFW=y
+CONFIG_EXAMPLES_CSIFW_TEST=y
+CONFIG_CSIFW_LOGS=y
+CONFIG_CSIFW_LOGE=y
+CONFIG_CSIFW_LOGI=y
```

Comment:
by default do you need all logs enabled. please check

Reply:
Done